apps/common/ThemeToggler.tsx

```diff
@@ -0,0 +1,17 @@
+import { Ionicons } from '@expo/vector-icons';
+import { TouchableOpacity, Platform } from 'react-native';
+
+import { useTheme } from './ThemeProvider';
+
+export function ThemeToggle() {
```

Comment:
ThemeToggle or ThemeToggler (as the file name)? Maybe it should be a default export?